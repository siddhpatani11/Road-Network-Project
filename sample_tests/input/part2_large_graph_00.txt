145 602
0 98 15 B
0 111 16 R
0 131 11 G
1 26 30 G
1 26 17 G
1 58 6 G
2 0 26 G
2 0 11 B
2 125 27 R
3 16 45 B
3 105 19 B
3 118 9 G
4 118 23 R
4 139 45 B
5 27 24 G
5 49 37 G
5 55 27 G
5 68 30 R
5 76 15 G
5 76 50 R
5 89 5 R
5 115 2 G
5 115 21 B
5 143 44 B
6 19 22 B
6 37 3 B
6 53 41 B
6 97 2 G
6 101 15 B
7 15 13 B
7 74 38 B
7 111 24 B
7 111 39 B
7 138 40 G
7 143 16 G
7 143 8 B
8 9 49 G
8 98 45 R
8 123 29 R
9 28 21 G
9 122 22 G
10 14 15 B
10 90 13 G
11 4 47 R
11 4 28 R
11 9 5 R
11 20 6 G
11 27 30 G
11 120 7 B
12 66 28 B
13 0 49 B
13 5 18 B
13 72 5 B
13 93 4 B
13 93 10 G
13 143 12 G
14 6 11 R
14 73 47 B
14 139 45 R
14 142 25 R
15 5 47 G
15 92 39 B
16 13 34 B
16 13 23 G
16 25 3 G
16 84 4 R
16 84 41 G
16 122 47 B
17 19 24 R
17 19 6 R
17 57 4 B
17 57 12 R
17 83 22 G
17 83 33 R
17 107 10 G
18 128 43 G
18 131 26 B
18 144 48 B
19 2 9 R
19 7 33 R
19 45 17 G
19 59 29 B
19 92 18 B
20 12 49 G
20 100 34 B
20 100 8 B
20 140 43 R
21 19 50 R
21 26 34 B
21 31 43 R
21 31 23 G
21 35 27 G
21 35 46 R
21 44 48 G
21 56 48 B
21 71 33 G
21 71 6 B
21 79 21 G
21 124 19 G
23 11 10 B
23 35 46 B
23 35 5 R
23 41 23 B
23 41 28 R
23 62 13 G
23 87 17 B
23 87 16 B
23 132 30 R
24 41 27 R
24 84 3 G
24 84 45 R
25 64 26 R
25 74 28 B
25 104 3 R
25 125 7 B
26 96 50 R
26 113 2 G
26 144 33 R
27 11 49 R
27 14 9 G
27 14 14 R
27 70 11 B
27 108 29 G
27 130 6 B
27 130 21 G
28 117 46 G
29 28 2 R
29 114 18 G
29 117 6 R
30 29 44 R
30 29 13 G
30 76 4 B
30 125 31 B
30 125 1 R
31 63 25 G
31 137 41 G
32 9 8 B
32 13 12 G
32 53 33 G
32 71 20 G
32 71 12 B
32 76 19 G
32 76 36 B
32 142 25 B
33 24 9 G
33 63 24 R
33 100 2 G
34 4 31 R
34 4 44 B
34 49 30 B
34 55 42 B
34 58 9 G
34 59 13 B
34 68 50 R
34 68 24 G
34 93 46 B
35 36 25 R
35 43 18 B
35 86 14 R
35 86 33 B
36 35 2 B
36 66 47 B
36 70 27 G
36 86 49 R
36 88 4 R
36 100 38 R
36 120 40 R
38 49 45 R
38 56 37 B
38 93 2 B
38 109 4 G
38 118 30 G
38 121 32 R
38 139 14 B
38 139 33 B
39 1 49 B
39 1 17 B
39 111 42 R
40 22 26 R
40 22 26 B
40 54 33 R
40 99 30 B
40 99 45 B
40 126 8 R
41 19 4 G
41 19 40 G
41 22 6 R
41 34 37 B
41 37 43 B
41 48 43 G
41 64 18 R
42 64 10 G
42 136 26 R
43 81 4 R
44 16 36 B
44 17 27 G
44 17 12 G
44 47 12 G
44 56 47 B
45 75 35 R
45 77 46 G
45 88 40 B
46 28 5 R
46 62 31 G
46 117 11 B
47 35 36 R
47 53 2 G
47 66 48 G
47 66 47 G
47 84 1 B
48 18 5 G
48 40 8 R
48 96 32 G
48 104 48 R
49 35 15 G
49 35 32 G
49 91 39 G
50 23 46 B
50 42 47 G
50 117 30 G
51 18 50 G
51 18 15 R
51 32 29 R
51 34 19 B
51 87 33 B
51 120 10 R
51 122 35 G
52 4 18 R
52 8 46 R
52 34 48 G
52 91 49 R
52 91 21 R
52 137 5 G
53 59 28 G
53 97 17 G
53 121 35 G
54 60 44 G
54 105 18 R
55 32 17 R
55 32 12 G
55 85 25 R
55 98 38 G
55 117 37 B
55 117 17 B
55 143 20 R
56 64 24 R
56 87 44 R
56 121 22 G
58 106 21 B
59 41 3 B
59 94 12 R
60 82 24 G
61 1 38 R
61 1 8 B
61 15 33 B
61 15 6 R
61 54 10 B
62 50 30 R
62 71 31 G
62 79 8 R
62 86 48 B
62 101 43 G
62 113 45 G
62 123 11 G
63 31 15 G
63 31 39 R
64 63 38 G
64 82 4 B
64 92 26 G
65 23 40 R
65 35 44 G
65 35 31 G
65 36 18 B
65 59 22 G
66 7 43 G
66 7 33 R
66 50 39 R
66 71 31 G
66 73 20 R
66 75 6 R
66 116 16 B
67 87 45 G
67 87 27 B
68 23 47 R
68 85 42 B
68 101 4 G
68 111 20 R
69 10 25 R
69 55 18 B
69 67 12 R
69 135 49 G
70 17 35 R
70 85 33 B
70 120 21 G
71 15 49 G
71 55 10 R
72 56 42 B
72 64 47 B
72 64 42 R
72 80 31 B
72 129 35 G
73 2 38 R
73 9 10 B
73 133 41 B
73 133 19 R
74 24 41 B
74 24 24 G
74 33 36 B
74 37 13 B
74 37 23 G
74 40 19 B
74 77 27 B
74 134 36 B
75 53 10 B
75 70 47 R
75 95 13 R
75 95 11 R
75 118 12 R
75 118 10 R
75 130 13 B
76 36 34 R
76 98 13 B
76 98 24 G
76 138 37 B
77 3 39 R
77 6 13 B
77 75 18 G
77 102 39 G
77 102 30 G
78 0 27 G
78 42 10 G
79 48 26 R
79 69 34 B
79 91 25 R
79 91 44 B
79 102 11 G
79 108 26 G
79 108 21 R
80 40 46 B
80 82 34 B
80 86 19 G
80 116 4 G
80 127 29 B
80 127 46 R
81 88 39 B
82 46 48 B
82 54 12 G
82 78 38 B
82 110 38 B
83 71 2 R
84 16 34 B
84 16 23 R
85 16 27 B
85 16 31 B
85 33 29 G
85 36 13 G
85 39 20 R
85 49 17 R
85 53 22 R
85 66 25 B
85 88 25 R
85 128 31 B
86 1 20 G
86 4 4 B
86 55 35 G
86 58 35 G
86 80 17 B
86 114 15 G
87 66 29 R
88 30 5 B
88 30 22 B
88 51 43 G
88 51 28 R
88 65 12 G
88 65 7 B
88 80 34 R
88 111 3 R
88 111 42 G
88 125 47 B
89 22 8 G
89 26 9 G
89 51 9 B
89 74 4 R
89 77 3 R
89 79 45 R
91 102 48 R
91 122 2 R
91 136 27 G
92 61 11 R
93 114 24 G
94 21 27 G
94 34 48 G
94 52 33 R
94 102 34 R
95 8 37 G
95 10 42 G
95 10 8 G
95 69 36 R
95 69 28 B
95 105 5 B
95 118 23 B
96 19 24 R
96 59 36 R
96 69 16 G
97 21 13 G
97 23 6 G
97 58 5 R
97 60 32 B
97 60 8 R
98 21 43 R
98 71 10 G
98 76 21 R
98 76 2 R
98 119 23 G
99 4 4 B
99 4 47 B
99 45 14 B
99 50 29 G
99 139 14 R
100 92 29 R
101 4 21 R
101 4 9 R
101 29 25 B
101 29 30 B
101 44 43 R
101 87 46 R
101 125 33 B
102 128 34 G
103 102 44 G
103 104 19 R
104 42 27 B
104 42 28 B
104 79 28 R
104 82 40 B
104 131 28 B
105 64 34 B
105 76 43 G
105 123 4 G
106 7 32 G
107 11 41 B
107 35 28 G
107 90 36 G
107 102 39 G
108 31 6 G
108 88 37 B
108 123 24 B
108 139 28 B
109 5 18 R
109 5 36 R
109 35 27 R
109 48 2 R
109 48 5 B
109 91 15 G
109 105 7 B
110 17 42 B
110 66 43 G
110 116 20 B
110 116 44 G
111 45 44 B
111 86 31 G
111 86 41 B
111 105 21 B
111 107 42 R
111 109 11 R
111 122 10 B
112 21 35 R
112 132 7 B
112 140 43 G
113 27 3 R
113 84 10 B
113 110 14 B
113 110 33 G
113 118 18 R
114 7 1 B
114 131 36 B
114 142 7 B
115 48 17 B
115 111 25 B
116 20 27 B
116 38 19 R
116 43 37 R
116 72 38 R
116 72 27 G
116 113 36 G
116 136 6 B
118 69 28 B
118 91 23 R
118 111 27 R
118 139 8 R
119 85 44 R
120 35 38 B
120 70 51 R
120 97 46 G
121 20 16 R
121 66 20 B
122 7 42 G
122 121 41 B
123 38 38 G
123 38 14 G
123 70 7 R
124 6 32 B
124 48 43 B
124 48 44 G
124 82 37 B
124 95 34 B
124 95 38 R
124 125 21 G
124 125 42 G
124 130 10 G
124 130 23 G
125 3 9 R
125 12 49 B
125 12 3 R
125 61 11 R
125 102 34 G
125 144 30 R
126 19 32 B
126 54 46 G
126 98 43 R
126 98 11 R
126 112 27 R
126 119 29 R
127 69 51 G
127 89 47 B
127 126 35 G
127 126 34 G
128 38 41 R
128 44 41 G
128 98 17 B
128 98 42 R
128 125 36 G
129 66 11 B
129 110 16 G
129 131 5 B
130 22 25 G
130 23 46 R
130 33 23 B
130 36 47 R
131 53 28 R
131 57 48 G
131 82 44 R
131 92 47 B
131 92 19 R
132 12 21 G
132 17 47 R
132 17 37 R
132 19 9 B
132 19 46 R
132 37 46 G
132 37 48 R
132 98 21 B
132 103 30 G
132 114 23 G
133 5 15 B
133 9 9 G
133 18 23 B
133 63 12 G
133 63 46 G
134 56 9 G
134 62 4 B
134 64 44 B
134 76 4 B
134 107 15 G
135 7 39 G
135 53 33 G
135 96 27 R
135 126 22 R
136 28 31 R
136 131 8 B
137 62 15 G
137 74 14 G
137 97 3 R
137 103 25 R
137 118 5 G
137 126 12 R
138 10 28 B
138 59 39 B
138 59 9 B
138 127 5 B
139 19 13 R
139 19 15 B
140 70 3 R
140 117 3 R
141 12 34 G
141 97 22 R
141 120 39 R
141 120 30 R
141 128 22 R
142 89 46 G
142 95 16 B
142 95 15 G
143 82 23 R
143 87 16 G
143 121 6 G
143 125 47 R
143 136 37 B
144 24 15 G
144 49 3 R
144 49 15 G
144 90 5 G
144 136 34 G
144 136 43 R
