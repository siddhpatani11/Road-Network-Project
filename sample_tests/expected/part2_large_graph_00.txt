51 R 90 R 343 R 275 B 115 B 50 R 59 R 155 B 105 - -1 G 316 B 177 - -1 B 109 R 41 - -1 B 118 R 188 - -1 R 15 R 266 G 92 R 104 R 72 - -1 B 151 - -1 R 73 - -1 B 100 B 98 - -1 R 127 R 29 R 323 B 19 G 116 R 307 - -1 - -1 R 414 R 174 G 135 G 245 G 134 R 152 R 505 - -1 - -1 R 166 G 101 G 198 - 0 - -1 R 62 G 132 R 269 R 199 - -1 R 358 G 132 R 88 B 117 - -1 R 126 R 88 R 431 B 62 B 164 B 69 B 152 R 361 R 49 R 46 G 179 B 143 G 165 R 48 - -1 - -1 R 125 R 154 G 138 G 120 - -1 R 192 R 294 R 137 B 33 R 419 R 64 B 141 G 140 G 95 R 45 G 253 R 157 G 364 R 56 R 61 - -1 R 487 B 73 B 194 - -1 R 214 R 482 R 379 B 171 - -1 B 140 - -1 R 106 - -1 - -1 R 338 - -1 - -1 B 104 B 124 - -1 R 10 G 76 G 35 - -1 R 123 B 106 R 234 R 200 R 58 - -1 - -1 G 76 - -1 - -1 - -1 R 242 G 141 - -1 R 85 - -1 - -1 - -1 G 333 B 113 G 98 
